ipatests/test_integration/test_ipa_ipa_migration.py

```diff
@@ -946,3 +1037,199 @@ def test_ipa_hbac_rule_duplication(self):
         count = Counter(line)
         assert count.get('Rule name: allow_all') < 2
         assert count.get('Rule name: allow_systemd-user') < 2
+
+    def test_ipa_migrate_check_service_status(self):
+        """
+        This testscase checks that ipactl and sssd
+        services are running post ipa-migrate tool
+        successful runs completed
+        """
+        cmd1 = self.replicas[0].run_command([
+            "ipactl", "status"
+        ])
+        assert cmd1.returncode == 0
+        cmd2 = self.replicas[0].run_command([
+            "systemctl", "status", "sssd"
+        ])
+        assert cmd2.returncode == 0
+
+    def test_custom_idrange_is_migrated(self, empty_log_file):
+        """
+        This testcase checks that custom idrange is migrated
+        from remote server to local server in production
+        mode.
+        """
+        range_name = "testrange"
+        CMD_OUTPUT = (
+            "---------------\n"
+            "1 range matched\n"
+            "---------------\n"
+            "  Range name: testrange\n"
+            "  First Posix ID of the range: 10000\n"
+            "  Number of IDs in the range: 10000\n"
+            "  First RID of the corresponding RID range: 300000\n"
+            "  First RID of the secondary RID range: 400000\n"
+            "  Range type: local domain range\n"
+            "----------------------------\n"
+            "Number of entries returned 1\n"
+            "----------------------------\n"
+        )
+        cmd = self.replicas[0].run_command(
+            ["ipa", "idrange-find", range_name])
+        assert CMD_OUTPUT in cmd.stdout_text
+
+    @pytest.fixture
+    def add_automount(self):
+        self.master.run_command(
```

Comment:
Move the preparation steps in prepare_ipa_server (add automountlocation, automountmap, automountkey) and remove this fixture + the call to this fixture in test_automountlocation_is_migrated.